aries-site/src/examples/foundation/accessibility/CorePrinciples.js

```diff
@@ -0,0 +1,95 @@
+import React, { useContext } from 'react';
+import {
+  Box,
+  Grid,
+  Image,
+  Paragraph,
+  ResponsiveContext,
+  Text,
+  ThemeContext,
+} from 'grommet';
+
+const object = [
```

Comment:
Naming related comment:
- Favor meaningful, descriptive variable names -- remember to think of the developers who will also access this code. 
  - When this is consumed below, how do I recall through what I am iterating?
  - What name might better represent the data here?